src/components/PoolingWidget/index.tsx

```diff
@@ -182,10 +184,15 @@ const PoolingInterface: React.FC = () => {
   // Avoid displaying an empty list of tokens when the wallet is not connected
   const fallBackNetworkId = networkId ? networkId : Network.Mainnet // fallback to mainnet
 
-  // TODO: switched to tagged tokens @anxo @leandro
-  const tokens = useMemo(() => tokenListApi.getTokens(fallBackNetworkId).filter(({ symbol }) => symbol !== 'WETH'), [
-    fallBackNetworkId,
-  ])
+  const tokens = useMemo(() => {
+    return (
+      // Get all the tokens for the current network
+      tokenListApi
+        .getTokens(fallBackNetworkId)
+        // Filter out the tokens not in the list
+        .filter(({ symbol }) => symbol && LIQUIDITY_TOKEN_LIST.includes(symbol))
```

Comment:
~Unnecessary~ small optimization, but `LIQUIDITY_TOKEN_LIST` would be better as a Set

Reply:
i agree!

Reply:
Done! 14a6190d2dc603d5b421ff22b4e6c6f32128030f